### DISCUSSION TITLE
Share what you're working on!

### DISCUSSION THREAD

**Discussion by andypiper** [andypiper/fivebyfive#4] [Show and tell]
I'd love to hear more about the ways the community is using this board...

- I think the [Wordle Device by @ciro](https://github.com/ccattuto/wordle-device) is really cool, standalone [Wordle](https://www.nytimes.com/games/wordle/index.html) results display from Twitter
- there is a nice [little clock display](https://github.com/didn0t/5x5_Clock) as well

I've got a lot of ideas currently listed in the notes file, happy to take contributions to this repo, or to link out to yours :-)

**Comment by RyeAlboa** [comment]
Hi - I’m totally new to coding and esp32 controllers, but I thought that I'd kicked off with the esp32-c3fh4 as a first project.

I’m looking to change the colour of the 5x5 nleds via an App & BLE - so an afternoons work right? 😂

**Comment by WouterVanGoey** [comment]
Hey Andy, I'm trying to flash WLED to this device but I am stuck at flashing the firmware + bootloader + partition table, I can't figure out which addresses to use, could you take a look? [I wrote about the issue](https://discord.com/channels/473448917040758787/473449548279185408/1275605546157735957) on the WLED discord. [Here's an invite if you need it.](https://discord.gg/nq5b8Vaxa9)
